packages/Calendar/styles.js

```diff
@@ -0,0 +1,197 @@
+import styled from 'styled-components'
+
+const theme = {
+  colors: {
+    telusPurple: '#4B286D',
+    shark: '#2A2C2E',
+    shuttleGrey: '#54595F',
+    white: '#FFF',
+    gainsboro: '#D8D8D8',
+  },
+  fontWeights: {
```

Comment:
These font weights can be be imported using `@tds/shared-typography@1.2.1` - https://github.com/telus/tds-core/blob/master/packages/typography/typography.js